stripecardscan/src/main/java/com/stripe/android/stripecardscan/cardscan/CardScanConfig.kt

```diff
@@ -0,0 +1,33 @@
+package com.stripe.android.stripecardscan.cardscan
+
+import androidx.annotation.Keep
+import com.stripe.android.camera.framework.time.seconds
+
+@Keep
+object CardScanConfig {
+
+    /**
+     * The duration after which the scan will reset if no card is visible.
+     */
+    @JvmStatic
+    var NO_CARD_VISIBLE_DURATION_MILLIS = 5.seconds.inMilliseconds.toInt()
+
+    /**
+     * The maximum duration for which to search for a card number.
+     */
+    @JvmStatic
+    var OCR_SEARCH_DURATION_MILLIS = 10.seconds.inMilliseconds.toInt()
```

Comment:
I'm not sure what the standard is for cardscan, but in payments we usually just add stuff like this to a companion object. Also if these are all internal we can mark it like this so that it doesn't become a part of our public API. I'm guessing users of our SDK won't need these.

```    
private companion object {
    private const val MYCONSTANT = "hey"
}
```

Reply:
These are in a centralized configuration because they are intended to be user configurable. Sometimes our users want to tune the scan specific to their user demographics.

I agree internal values should be in companion objects, but these are intended to be external.

Reply:
hmm interesting. What's the advantage of using this centralized configuration versus passing it as part of a constructor? It seems like this would let users change things at varying times. Is that desirable?

Reply:
That's true. It's unlikely that a user would modify these values mid-scan, but it's possible to do so. Given the values in this configuration, nothing would break if they were changed mid-scan, but it would potentially be a little strange for the user.

That said, I think you're right about this pattern being generally undesirable. We should probably pass a `ScanConfiguration` object in as part of the `CardScanActivityParams` instead of having a global configuration object.

However, this feels a little outside the scope of this PR. let's consider it for a follow-up task to move all of the configuration into a constructor parameter.